test/integrationTests/hoverProvider.integration.test.ts

```diff
@@ -48,15 +48,11 @@ namespace Test
        }
    }
 }`;
-        let fileUri = await testAssetWorkspace.projects[0].addFileWithContents("test1.cs", program);
-
+        let fileUri = await testAssetWorkspace.projects[0].addFileWithContents("hover.cs", program);
         await omnisharp.restart();
         await omnisharp.waitForEmptyEventQueue();
-
-        await vscode.workspace.openTextDocument(fileUri);
-
+        await vscode.commands.executeCommand("vscode.open", fileUri);
```

Comment:
Why did you move away from the openTextDocument helper that you switched to?

Reply:
No specific reason. I just thought it would maintain consistency with the codeActionRename test. Is openTextDocument a better way ?